### PR TITLE
update getting started lib docs with note on compat

### PR DESCRIPTION
**Pull request by smackesey**
### Summary & Motivation

Inspired by this [slack thread](https://elementl-workspace.slack.com/archives/C03AFBP4QQ2/p1659729486640719)

### CODE REVIEW

**Comment by smackesey** [comment]
Current dependencies on/for this PR:
* master
  * **PR #8965** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/8965" target="_blank">Graphite</a> 
    * **PR #8960** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/8960" target="_blank">Graphite</a> 
      * **PR #8961** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/8961" target="_blank">Graphite</a> 
      * **PR #9246** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/9246" target="_blank">Graphite</a> 
      * **PR #9260** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/9260" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/9260?utm_source=stack-comment).

File: docs/content/getting-started/releases.mdx [hunk 1]
```diff
@@ -33,7 +33,9 @@ Like non-deprecated public stable APIs, deprecated public stable APIs will not b
 
 ## Dagster integration libraries
 
-Dagster's integration libraries are in the 0.y.z of [Semantic Versioning](https://semver.org/). While technically this is "anything goes", we are conservative about making changes and provide guidance about when to expect breaking changes:
+Dagster’s integration libraries haven’t yet achieved the same API maturity as Dagster core. For this reason, integration libraries remain on a pre-1.0 versioning track (in general 0.y.z of [Semantic Versioning](https://semver.org/); and 0.16+ as of Dagster 1.0.0) for the time being. However, 0.16+ library releases remain fully compatible with Dagster 1.x. In the coming months, we will graduate integration libraries one-by-one to the 1.x versioning track as they achieve API maturity.
```
Comment (dpeng817) [inline docs/content/getting-started/releases.mdx:36]: I'd remove the `in the coming months piece`.

**Review by dpeng817 — APPROVED** [review]
one comment

File: docs/content/getting-started/releases.mdx (hunk 1 above)
Comment (smackesey) [reply docs/content/getting-started/releases.mdx]: good call

**Pull request merged**